docs/content/en/tasks/manage/private-registries/_index.md

```diff
@@ -0,0 +1,393 @@
+    tag:
+    pullPolicy: 
+```
+
+Example YAML:
+
+```yaml
+cassandra:
+  serviceAccount: cassandra
+  image:
+    registry: myregistry
+    repository: myrepo/cass-management-api
+    tag: 3.11.10-v0.1.26
+    pullPolicy: IfNotPresent
+```
+
+{{% alert title="Tip" color="success" %}}
+While `pullPolicy` is shown above, it is not part of the image coordinates. Possible values for `pullPolicy` are:
+
+* `IfNotPresent` (default)
+* `Always` (default when the image tag is `latest`)
+* `Never`
+
+See [ImagePullPolicy](https://helm.sh/docs/chart_best_practices/pods/#imagepullpolicy) in the Helm v3 documentation.
+{{% /alert %}}
+
+### List of chart properties
+
+Here is a list of chart properties that correspond to K8ssandra deployed containers having an `image` property:
+
+| Chart property / container name                          | Summary                               |
+| -------------------------------------------------------- | ------------------------------------- |
+| `cassandra`                                              | Runs the Management API and Cassandra&reg; |
+| `cassandra.configBuilder`                                | An init container that provisions Cassandra configuration files such as cassandra.yaml |
+| `cassandra.jmxCredentialsConfig` | An init container that configures authentication for remote JMX access |
+| `cassandra.loggingSidecar` | A container that tails Cassandra's system.log file |
+| `stargate` | Runs Stargate, which is an API for interacting with Cassandra |
+| `stargate.waitForCassandra` | An init container that waits until all the Cassandra pods are ready |
+| `reaper` | Run Reaper, which provides repair functionality for Cassandra data |
+| `medusa` | Runs the medusa container that performs backups, and the `medusa-restore` init container that performs restores |
+| `cleaner` | A Helm pre-delete hook that deletes the CassandraDatacenter |
+| `client` | Used in the `crd-upgrader pre-upgrade` to apply Custom Resource Definition (CRD) updates |
+| `cass-operator` | Cass Operator |
+| `reaper-operator` | Reaper Operator |
+| `medusa-operator` | Medusa Operator |
+| `kube-prometheus-stack.prometheus-operator` | Prometheus Operator |
+| `kube-prometheus-stack.prometheus-operator.prometheusSpec` | Prometheus to collect Cassandra cluster, OS, and node metrics |
+| `grafana` | Grafana to visualize the metrics collected by Prometheus |
+
+{{% alert title="Tip" color="success" %}}
+A general note: Operators are software extensions to Kubernetes that make use of custom resources to manage applications and their components. Thus, for example, "Reaper Operator" deploys and configures Reaper. "Reaper" itself manages the actual Cassandra repair operations. Similarly, "Prometheus Operator" deploys and configures Prometheus. "Prometheus" itself manages the actual collection of relevant OS / Cassandra metrics. "Medusa Operator" deploys and configures Medusa. "Medusa" itself manages the backup/restore of Cassandra data. 
```

Comment:
>"Medusa Operator" deploys and configures Medusa. "Medusa" itself manages the backup/restore of Cassandra data.

Medusa Operator is a bit atypical compared to the other operators in that it does not deploy and configure Medusa. It configures and orchestrates the backup and restore operations.

Reply:
Updated in subsequent commit - thanks John.